lib/printer.go

```diff
@@ -219,11 +221,16 @@ func diffPrinter(pn, pg *Printer) PrinterDiff {
 		d.DailyQuotaChanged = true
 	}
 
+	if pg.NotificationChannel != pn.NotificationChannel {
+		d.NotificationChannelChanged = true
+	}
+
 	if d.DefaultDisplayNameChanged || d.ManufacturerChanged || d.ModelChanged ||
-		d.GCPVersionChanged || d.SetupURLChanged || d.SupportURLChanged ||
-		d.UpdateURLChanged || d.ConnectorVersionChanged || d.StateChanged ||
-		d.DescriptionChanged || d.CapsHashChanged || d.TagsChanged ||
-		d.DuplexMapChanged || d.QuotaEnabledChanged || d.DailyQuotaChanged {
+			d.GCPVersionChanged || d.SetupURLChanged || d.SupportURLChanged ||
```

Comment:
looks like some unnecessary whitespace changes in these lines?

Reply:
they are auto formatted.